src/geo/gmaps/gmaps-map-view.js

```diff
@@ -103,13 +103,22 @@ var GoogleMapsMapView = MapView.extend({
   _setScrollWheel: function (model, z) {
     this._gmapsMap.setOptions({ scrollwheel: z });
   },
-
-  _setZoom: function (model, z) {
+  _setView: function (model) {
+    if (model.changed) {
+      if (model.changed.zoom) {
```

Comment:
Take a look at http://backbonejs.org/#Model-hasChanged. Perhaps we can use it here.